question-generation/nlp.py

```diff
@@ -0,0 +1,42 @@
+from transformers import pipeline
+
+
+def interview_ai(sequence: str, num_questions: int) -> "list[str]":
```

Comment:
function name?

Reply:
It could be better– I couldn't think about a good one while writing this. Do you have a suggestion?

Reply:
`generate_interview_questions`?